### DISCUSSION TITLE
How do I edit the main login page?

### DISCUSSION THREAD

**Discussion by YaffarOmarOZIL** [Laravel-Backpack/community-forum#1149] [Q&A (Help)] [answered]
![imagen](https://github.com/user-attachments/assets/106b28b4-86a4-4b98-ba76-a8bfea5076dc)

It may be something super simple to achieve but I'm just starting with this and I wanted to know if it's possible and if it is, how can I get the view or the code where's the page / styles files, and if possible edit them.
I tried with:
`php artisan backpack:publish`
but I don't know what the files are :(
Is there any possibility or in the paid function perhaps?

**Comment by jcastroa87** [comment, marked as answer]
Hello @YaffarOmarOZIL 

As i see you are using Tabler, in our [README](https://github.com/Laravel-Backpack/theme-tabler?tab=readme-ov-file#overriding), there are instructions to override views.

Cheers.

**Reply by YaffarOmarOZIL** [reply]
Thank you very much :3
